data/legend/metadata/datasets/statuses/l200-p02-r009-T%-all-config.yaml

```
@@ -0,0 +1,6 @@
+V99000A:
+    processable: true
+    usability: "on"
+B99000A:
+    processable: true
+    usability: "off:
```

This breaks our code
```suggestion
    usability: "off"
```